client/src/components/new-request/analyze-results.vue

```diff
@@ -1,29 +1,38 @@
 <template>
   <MainContainer id="analyze-results-container">
     <template v-slot:container-header>
-      <v-col cols="auto" class="h4">
+      <v-col cols="auto" class="font-weight-bold h6 ml-3">
         You are searching for a name for a
         {{ entityText === ' BC Corporation' && location.text === ' BC' ? '' : ' ' + location.text }} {{ entityText }}
       </v-col>
     </template>
     <template v-slot:content>
       <v-row>
-        <v-col cols="12" class="mt-3" @click="clickNameField">
+        <v-col cols="12" class="mt-3 pb-0" @click="clickNameField">
           <quill-editor :contents="contents"
                         :options="config"
                         :disabled="!!finalName"
                         @change="handleChange($event)"
                         @keydown.native.capture="handleEnterKey"
                         id="name-search-bar"
                         ref="quill" />
-          <div style="position: relative; left: 865px; top: -45px; z-index: 1000">
-            <v-icon @click.capture.stop="handleSubmit"
-                    class="name-search-icon"
-                    id="name-input-icon">mdi-magnify</v-icon>
+          <div style="position: relative; left: 96.5%; top: -45px; z-index: 1000">
```

Comment:
I know this is existing code but I highly recommend this be extracted into a new CSS class.

Reply:
Sounds good, will do!

Reply:
Updated!